### PR TITLE
Remove OrderedSet

### PR DESCRIPTION
**Pull request by LukasHromadnik**
* Bumped deployment target to 12.0. Our projects are >= 12.
* Continue refactoring of SSCheckMark
* Removed OrderedSet since we had some crashes in that thing

#### Checklist
- [x] Added tests (if applicable)

### CODE REVIEW

File: ACKImagePicker/ViewControllers/ACKPhotosViewController.swift [hunk 1]
```diff
@@ -241,21 +241,17 @@ extension ACKPhotosViewController: UICollectionViewDataSource {
...
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        guard let object = fetchResult?.object(at: indexPath.item) else { assertionFailure(); return }
-
         if let maxNumberOfImages = delegate?.maximumNumberOfSelectedImages, selectedImages.count >= maxNumberOfImages {
             collectionView.deselectItem(at: indexPath, animated: false)
             return
         }
 
-        selectedImages.add(object)
+        selectedImages.append(indexPath.item)
```
Comment (olejnjak) [inline ACKImagePicker/ViewControllers/ACKPhotosViewController.swift:253]: Do we know that array will work correctly without checking for duplicates?

**Review by olejnjak — APPROVED** [review]

File: ACKImagePicker/ViewControllers/ACKPhotosViewController.swift (hunk 1 above)
Comment (RomanPodymov) [reply ACKImagePicker/ViewControllers/ACKPhotosViewController.swift:253]: Or you can use https://github.com/apple/swift-collections

Comment (olejnjak) [reply ACKImagePicker/ViewControllers/ACKPhotosViewController.swift:253]: Yes, but I don't think it is worth adding a dependency..

Comment (LukasHromadnik) [reply ACKImagePicker/ViewControllers/ACKPhotosViewController.swift:253]: I've added `contains` check just to be sure. From testing everything was working correctly but no one can be 100 % sure 🤓

**Pull request merged**